MovieSharing/VideoModel.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  Video.swift
+//  MovieSharing
+//
+//  Created by Mariam Korashi on 28.09.21.
+//
+
+import Foundation
+
+struct VideosResponse:Decodable {
```

Comment:
code style comment: please leave a single space between the name of the class/struct/var etc and its type